### PR TITLE
remove last value

### PR DESCRIPTION
**Pull request by vlad12345123**
remove last value from summaries array

### CODE REVIEW

File: InternshipBE/WebApi/Controllers/WeatherForecastController.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ public class WeatherForecastController : ControllerBase
     {
         private static readonly string[] Summaries = new[]
         {
-            "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+            "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering"
         };
```
Comment (artemgael96) [inline InternshipBE/WebApi/Controllers/WeatherForecastController.cs:17]: Why did you delete?

Comment (vlad12345123) [reply InternshipBE/WebApi/Controllers/WeatherForecastController.cs:17]: Because its my example working with git

Comment (artemgael96) [reply InternshipBE/WebApi/Controllers/WeatherForecastController.cs:17]: ok you are getting a decline your pull request

Comment (artemgael96) [reply InternshipBE/WebApi/Controllers/WeatherForecastController.cs:17]: ok, you are getting a decline your pull request

**Comment by artemgael96** [comment]
it was a test of pull request
